packages/require/src/require.js

```diff
@@ -131,15 +179,23 @@ export function require(urlOrSource, transform, _readFile, _base, root, readModu
     }
   }
   if (source !== undefined) {
-    // do not transform bundles that are already cjs ( requireCache.bundleAlias.*)
-    if (transform && !bundleAlias) source = transform(source, url).code
-    let requireFunc = newUrl => require(newUrl, transform, readFile, url, root, readModule, moduleBase)
-    const module = requireModule(url, source, requireFunc)
-    module.local = isRelativeFile
-    exports = module.exports
+    let extension = getExtension(url)
+    // https://cdn.jsdelivr.net/npm/@jscad/svg-serializer@2.3.13/index.js uses require to read package.json
+    if(extension === 'json'){
+      exports = JSON.parse(source)
+    }else{
+      // do not transform bundles that are already cjs ( requireCache.bundleAlias.*)
+      if (transform && !bundleAlias) source = transform(source, url).code
+      let requireFunc = newUrl => require(newUrl, transform, readFile, base, root, readModule, moduleBase)
+      const module = requireModule(url, source, requireFunc)
+      module.local = isRelativeFile
+      exports = module.exports
+    }
   }
-
-  if (typeof exports !== 'object') {
+  
+  // allow aslo function workaround for weid case in @jscad/io-utils/Blob.js
+  if (typeof exports !== 'object' && typeof exports !== 'function') {
+    console.error('exports',typeof exports, exports)
     throw new Error(`module ${cacheUrl} did not export an object`)
   }
```

Comment:
Technically, you could also do things like:
```js
module.exports = 'export a string'
module.exports = 3.14
```

What's interesting though is that if you load say, an empty file, then exports will actually be empty object `{}`. So I don't think this entire check is doing... anything? We handle the missing main case in `worker.js:51`. I propose we just delete this whole if block?

```suggestion
```